components/Layout/Account/LayoutAccountOverviewContent.vue

```diff
@@ -0,0 +1,84 @@
+<script setup lang="ts">
+const customerStore = useCustomerStore();
```

Comment:
```suggestion
const customerStore = useCustomerStore();
const { customer } = storeToRefs(customerStore)
```

you are doing a lot of `customerStore.customer.<something>` below that shortens it a little bit
also you can probably to this:
```
const billingAddress = computed(() => customer.value.defaultBillingAdress);
```
For billing address as well als the shipping and payment methods also making the template a little more clean

Reply:
Good point! I included this in #47.